redcap_hooks.php

```diff
@@ -155,4 +175,4 @@ function redcap_user_rights($project_id = null) {
 
 /////////////////////////////////////
 
```

Comment:
Did you mean to comment this out, or remove it?

Reply:
I intended to remove it because the error levels in REDCap are all weird and this has been filling the logs with garbage for years without value.